pkg/master/master_test.go

```diff
@@ -343,6 +342,45 @@ func TestDiscoveryAtAPIS(t *testing.T) {
 	assert.Equal(expectGroupName, groupList.Groups[0].Name)
 	assert.Equal(expectVersions, groupList.Groups[0].Versions)
 	assert.Equal(expectPreferredVersion, groupList.Groups[0].PreferredVersion)
+
+	master.thirdPartyResources["/apis/company.com/v1"] = thirdPartyEntry{
+		nil,
+		unversioned.APIGroup{
+			Name: "company.com",
+			Versions: []unversioned.GroupVersionForDiscovery{
+				{GroupVersion: "company.com/v1", Version: "v1"},
+			},
+			PreferredVersion: unversioned.GroupVersionForDiscovery{GroupVersion: "company.com/v1", Version: "v1"},
```

Comment:
You are initializing the same unversioned.GroupVersionForDiscovery 4 times. You can initialize it once and reuse it at other places?